### PR TITLE
Issue 28563 removing template ajax and js deps

### PR DESCRIPTION
**Pull request by jdotcms**
First draft to remove the templateAjax

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateResource.java [hunk 1]
```diff
@@ -802,4 +805,52 @@ public final Response delete(@Context final HttpServletRequest  request,
                 new BulkResultView(deletedTemplatesCount,0L,failedToDelete)))
                 .build();
     }
+
+    /**
+     * Return live version {@link com.dotmarketing.portlets.templates.model.Template} based on the id
```
Comment (jcastro-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateResource.java:810]: Javadoc doesn't seem to match the method's definition, or it's name is not right.

**Review by jcastro-dotcms — CHANGES_REQUESTED** [review]

**Review by dsolistorres — APPROVED** [review]

**Review by freddyDOTCMS — COMMENTED** [review]
postman test?

**Review by freddyDOTCMS — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java [hunk 2]
```diff
@@ -1553,4 +1553,11 @@ public void onCopySite(final SiteCreatedEvent event)
 		APILocator.getFolderAPI().subscribeFolderListener(appTemplateFolder, new ApplicationTemplateFolderListener(),
 				childName -> null != childName && (childName.endsWith(Constants.VELOCITY_FILE_EXTENSION) || childName.endsWith(Constants.JSON_FILE_EXTENSION)));
 	}
+
+	@CloseDBIfOpened
+	@Override
+	public Contentlet getImageContentlet(final Template template) throws DotDataException, DotSecurityException {
```
Comment (wezell) [inline dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java:1559]: Probably should return an optional

**Review by wezell — APPROVED** [review]
Left a comment but still approving.

**Review by freddyDOTCMS — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java (hunk 2 above)
Comment (jdotcms) [reply dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java]: feedback done, thx

**Review by wezell — APPROVED** [review]
Nice stuff!

**Review by jcastro-dotcms — APPROVED** [review]

**Pull request merged**